pgpro_tam/benchmark.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+#Usage:
+#./benchmark.sh parquet_fd
```

Comment:
The invocation is is too confusing what is being measured here. What does "fd" mean? I first thought it means "feather" but there is a separate feather invocation (l. 7). If "fd" occurs in all four invocations (l. 4-7), why do we need to specify it at all?

Reply:
pgpro_tam works with multiple formats and storage managers. 'fd' states for ppg_fd_smgr and points to classic 'fd.c' from postgres, which implements basic operations with storage. There is another storage manager - ppg_mem_fd_smgr, which extends possibilities of the previous one. Keeping only 'mem' for this may cause misunderstanding, as it's not a stateless in-memory storage.
If it makes sense, they can be renamed, but anyway, results and readme already contain a human-readable description.